### PR TITLE
Bump to 0.66.0, update changelog

### PR DESCRIPTION
**Pull request by ehuss**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,20 +1,75 @@
...
+  [`workspace.package`](https://doc.rust-lang.org/nightly/cargo/reference/workspaces.html#the-workspacepackage-table)
+  and
+  [`workspace.dependencies`](https://doc.rust-lang.org/nightly/cargo/reference/workspaces.html#the-workspacedependencies-table)
+  for more details on how to define these common settings.
+  [#10859](https://github.com/rust-lang/cargo/pull/10859)
+- Added the
+  [`--crate-type`](https://doc.rust-lang.org/nightly/cargo/commands/cargo-rustc.html#option-cargo-rustc---crate-type)
+  flag to `cargo rustc` to override the crate type.
+  [#10838](https://github.com/rust-lang/cargo/pull/10838)
+- Cargo commands can now accept multiple `--target` flags to build for
```
Comment (weihanglo) [inline CHANGELOG.md:35]: Is it worth to mention `build.target` accepting an array as well?

**Review by weihanglo — COMMENTED** [review]
Thank you for this update!

File: CHANGELOG.md (hunk 1 above)
Comment (ehuss) [reply CHANGELOG.md:35]: Sure!

**Comment by weihanglo** [comment]
@bors r+

Thanks!

**Pull request merged**